addons/compact-messages/addon.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Compact Messages",
+  "description": "Makes the messages page compact and easy to read.",
+    "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/messages"]
+    }
+  ],
+  "tags": ["messages", "compact"],
```

Comment:
Tags are not custom, you have to pick one or two from the list.

Reply:
which list

Reply:
[This list](https://scratchaddons.com/docs/reference/addon-manifest/#tags-array-required)

Reply:
i didn't read that